platform/stm32f4_cnc/http_admin/http_admin_backend.c

```diff
@@ -63,30 +63,30 @@ static char *http_admin_build_iface_list(void) {
 
 		cJSON_AddStringToObject(iface_obj, "ip",
 				inet_ntop(iaddr->sa_family,
-					&((struct sockaddr_in *) iaddr)->sin_addr,
-					buf,
-					sizeof(buf)));
+				&((struct sockaddr_in *) iaddr)->sin_addr,
+				buf,
+				sizeof(buf)));
 
 		cJSON_AddStringToObject(iface_obj, "netmask",
 				inet_ntop(i_ifa->ifa_netmask->sa_family,
-					&((struct sockaddr_in *) i_ifa->ifa_netmask)->sin_addr,
-					buf,
-					sizeof(buf)));
+				&((struct sockaddr_in *) i_ifa->ifa_netmask)->sin_addr,
+				buf,
+				sizeof(buf)));
 
 		iface_dev = inetdev_get_by_name(i_ifa->ifa_name);
 		if (!iface_dev) {
 			goto outerr;
 		}
 		macaddr_print((unsigned char *) buf, (unsigned char *) iface_dev->dev->dev_addr);
 		cJSON_AddStringToObject(iface_obj, "mac", buf);
-        }
+	}
 
 	json_list = cJSON_PrintUnformatted(iface_array);
 
 	cJSON_Delete(iface_array);
 	freeifaddrs(ifa);
 	return json_list;
-outerr:
+	outerr:
```

Comment:
labels should not be indented (something desired is https://www.kernel.org/doc/html/v4.10/process/coding-style.html#centralized-exiting-of-functions)